<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>788</width>
    <height>593</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>家庭智慧中控系统</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>191</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华光黑变_CNKI</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>空调管理</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>空调.png</normaloff>空调.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>401</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="pushButton_5">
       <property name="font">
        <font>
         <family>华光黑变_CNKI</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>关机</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>空调.png</normaloff>空调.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_6">
       <property name="font">
        <font>
         <family>华光黑变_CNKI</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>开机</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>空调.png</normaloff>空调.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_7">
       <property name="font">
        <font>
         <family>华光黑变_CNKI</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>连接配置</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>空调.png</normaloff>空调.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton_14">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>20</y>
      <width>191</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华光黑变_CNKI</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>门禁管理</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>门禁管理.png</normaloff>门禁管理.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_17">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>340</y>
      <width>221</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华光黑变_CNKI</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>家庭卫士</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>卫士.png</normaloff>卫士.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>90</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_21">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>230</y>
      <width>191</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华光黑变_CNKI</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>光照管理</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>灯泡.png</normaloff>灯泡.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>401</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="font">
        <font>
         <family>华光黑变_CNKI</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>温度+</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>空调.png</normaloff>空调.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_4">
       <property name="font">
        <font>
         <family>华光黑变_CNKI</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>模式</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>空调.png</normaloff>空调.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_3">
       <property name="font">
        <font>
         <family>华光黑变_CNKI</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>温度-</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>空调.png</normaloff>空调.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>110</y>
      <width>331</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="pushButton_15">
       <property name="font">
        <font>
         <family>华光黑变_CNKI</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>开门</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>门禁管理.png</normaloff>门禁管理.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_16">
       <property name="font">
        <font>
         <family>华光黑变_CNKI</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>锁门</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>门禁管理.png</normaloff>门禁管理.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>430</y>
      <width>591</width>
      <height>100</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="pushButton_18">
       <property name="font">
        <font>
         <family>华光黑变_CNKI</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>启动</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>卫士.png</normaloff>卫士.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>90</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_19">
       <property name="font">
        <font>
         <family>华光黑变_CNKI</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>重置</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>卫士.png</normaloff>卫士.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>90</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_20">
       <property name="font">
        <font>
         <family>华光黑变_CNKI</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>门禁查看</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>卫士.png</normaloff>卫士.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>90</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>788</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
